docs/application/dotnet/guides/uix/tts.md

```diff
@@ -669,6 +697,40 @@ To start, pause, and stop the playback, follow the steps below:
     }
     ```
 
+<a name="playing_mode"></a>
+## Playing mode
+
+Playing mode determines which process, either TTS service or TTS client, plays the synthesized PCM data. Described below are the 2 different types of playing modes:
+
+> [!NOTE]
+> Playing mode is different from TTS mode. Playing mode only determines a subject to play PCM data.
+
+- `ByService`: TTS service plays the synthesized PCM data. If you do not set playing mode, it will be set as `ByService`.
+- `ByClient`: TTS client receives the synthesized PCM data from TTS service, and plays it directly.
+
+To decide between the client-side playback mode and the service-side playback mode, follow the steps below:
+
+-   Set the playing mode
+
+    If the application wants to play the PCM data directly instead of playing it that the application requested for synthesis in the TTS service, you can set the playback mode to `ByClient`. If the playback mode is not set by the user, the TTS service will synthesize the text and play it using the default value `ByService`.
+
+    > [!NOTE]
+    > If the playing mode is `ByService`, you don't need to set a event handler. It will be played automatically by the TTS service. If you set the mode to `ByClient`, you can receive the synthesized PCM data via `SynthesizedPcm` event and only change playing mode when TTS client state is `Created`. If you want to use both playing modes, it would be better to create 2 TTS handles:
```

Comment:
```suggestion
    > If the playing mode is `ByService`, you don't need to set an event handler. It will be played automatically by the TTS service. If you set the mode to `ByClient`, you can receive the synthesized PCM data via `SynthesizedPcm` event and only change playing mode when TTS client state is `Created`. If you want to use both playing modes, it would be better to create 2 TTS handles.
```

Reply:
Thank you for checking. I will fix it soon.